OpenUpSA/wazimap-ng-ui
# Updated filter comparison to show snackbar

## Description
Updated how we are comparing filters

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-711

## How is it tested automatically?


## How should a reviewer test it locally
* Make sure we have non aggregatable filters
* When we select indicator with non aggregatable filters it should show snackbar on first subsindicator click but should not show snackbar when other subsindicator with same filters are clicked

## Screenshots


## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
